mopro-ios/MoproKit/Example/MoproKit/MSMBenchmarkViewController.swift

```diff
@@ -10,7 +10,26 @@ import UIKit
 import MoproKit
 
 class MSMBenchmarkViewController: UIViewController, UITableViewDelegate, UITableViewDataSource {
+    
+    // let pointUrl = URL(string: "https://mopro-msm.s3.eu-north-1.amazonaws.com/vectors/16x10/points")
+    // let scalarUrl = URL(string: "https://mopro-msm.s3.eu-north-1.amazonaws.com/vectors/16x10/scalars")
+    // func runDownloadAction() {
+    //   let pointStart = CFAbsoluteTimeGetCurrent()
+    //   FileDownloader.loadFileAsync(url: self.pointUrl!) { (path, error) in
+    //     print("Points File downloaded to : \(path!)")
+    //     let pointEnd = CFAbsoluteTimeGetCurrent()
+    //     print("Download points took:", pointEnd - pointStart, "s")
+    //   }
+
+    //   let scalarStart = CFAbsoluteTimeGetCurrent()
+    //   FileDownloader.loadFileAsync(url: self.scalarUrl!) { (path, error) in
+    //     print("Scalars File downloaded to : \(path!)")
+    //     let scalarEnd = CFAbsoluteTimeGetCurrent()
+    //     print("Download scalars took:", scalarEnd - scalarStart, "s")
+    //   }
 
+    // }
```

Comment:
If we can confirm that the benchmarking instances can always be generated on both iOS simulator and mobile device, we should consider removing these comments in the later PR.